pkg/render/funcs.go

```diff
@@ -28,3 +32,9 @@ func isSet(m map[string]interface{}, key string) interface{} {
 	}
 	return val
 }
+
+// escapeCharacters returns the given string with any
+// possible reference of '$' escaped.
+func escapeCharacters(text string) string {
+	return strings.ReplaceAll(text, "$", "\\$")
```

Comment:
If it's so simple, you could probably just use sprig function [1]. But maybe it's better to leave it as a separate function as we might discover more special cases. 

[1] https://masterminds.github.io/sprig/strings.html#replace

Reply:
yup